### PR TITLE
Add single cell section

### PR DESCRIPTION
**Pull request by bebatut**
We will probably have more and more single-cell tools in the future. So having a dedicated section may make sense

Let me know what you think about that

### CODE REVIEW

**Comment by erasche** [comment]
Sounds like a good idea to me.

File: files/galaxy/config/tool_conf.xml [hunk 1]
```diff
@@ -276,6 +276,8 @@
   </section>
   <section id="mimodd" name="MiModD">
   </section>
+  <section id="singlecell" name="Single-cell" >
```
Comment (bgruening) [inline files/galaxy/config/tool_conf.xml]: Should this be under RNA analysis?

Comment (erasche) [reply files/galaxy/config/tool_conf.xml]: If there are enough tools, in a coherent suite/toolkit, then could make sense to have separate. If you'd rather they belong under RNA then could work too.

Comment (bgruening) [reply files/galaxy/config/tool_conf.xml]: Not under RNA section, sorry but under the entry ... mh, ok again. I think this should be under/in "Genomics Analysis" rather then "Genomics Toolkits"

Comment (erasche) [reply files/galaxy/config/tool_conf.xml]: +1

Comment (bebatut) [reply files/galaxy/config/tool_conf.xml]: Oh I thought it was still Genomics Analysis. I missed the new label
Fix it in #37

**Pull request merged**